### ISSUE TITLE
Consommation de viande en poids plutôt qu'en repas

### ISSUE THREAD

**Issue by publicodes** [incubateur-ademe/nosgestesclimat#961]
Ne pourrait-on pas parler de kg de viande consommée par année ou mois ? manger 30gr ou 300gr sur un repas n'est pas la même chose

Depuis la page : `https://nosgestesclimat.fr/documentation/alimentation/réduire-viande/par-deux`

> Ce ticket a été créé automatiquement par notre robot depuis notre [page de contribution](https://nosgestesclimat.fr/contribuer).

**Comment by laem** [comment]
Oui, c'est c'est une possibilité, mais ça rend la saisie du formulaire bien plus compliquée. Bénéfice : précision; Risque : perdre beaucoup d'utililsateurs qui ne mesurent pas leurs portions. C'est en particulier le cas en restaurant d'entreprise, cantine et restaurant tout court.

Si l'on considère aujourd'hui qu'on mange une petite portion de viande par jour plutôt qu'une portion classique, il est toujours possible dans l'interface actuelle de "déclarer" moins de repas viande. On pourra réfléchir à une version plus élaborée de la simulation, mais il faudra trouver des idées de design.

**Comment by avenirclimatique** [comment]
Je confirme. La saisie au poids est possible en utilisant l'outil MicMac d'Avenir Climatique (en tableur Excel ou en ligne), mais nous nous sommes rendu compte à l'usage que c'est très peu pratique. La plupart du temps, les utilisateurs ne connaissent pas la masse de produits consommés, et lorsqu'elle est estimée, c'est en général une sous-estimation.

**Comment by JuliePouliquen** [comment]
Par curiosité @Clemog tu l'as gardée ouverte en vue d'un jour revoir la sélection des repas, c'est bien cela ?

**Comment by Clemog** [comment]
Plutôt pour conserver l'historique de "pourquoi l'actuel"
